.github/workflows/docker-publish.yml

```diff
@@ -16,10 +16,6 @@ jobs:
       -
         name: Checkout
         uses: actions/checkout@v3
```

Comment:
According to the [`docker/build-push-action` docs](https://github.com/docker/build-push-action#git-context), the "Checkout" step can also be removed from this workflow:

> By default, this action [`docker/build-push-action`] uses the [Git context](https://docs.docker.com/engine/reference/commandline/build/#git-repositories), so you don't need to use the [actions/checkout](https://github.com/actions/checkout/) action to check out the repository as this will be done directly by [BuildKit](https://github.com/moby/buildkit).

```suggestion

```